SneakPeek/res/layout/sneaker_entry_listitem.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent" >
+
+    <ImageView
+        android:id="@+id/imgSneakerThumbnail"
+        android:layout_width="@dimen/sneaker_thumbnail_h_w"
+        android:layout_height="@dimen/sneaker_thumbnail_h_w"
+        android:layout_alignParentBottom="true"
+        android:layout_alignParentLeft="true"
+        android:layout_alignParentTop="true"
+        android:layout_gravity="center"
+        android:gravity="center_vertical"
+        android:src="@drawable/jordanivundefeated" />
+
+    <TextView
+        android:id="@+id/txtShoeName"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:layout_alignParentTop="true"
+        android:layout_marginLeft="@dimen/sneaker_txt_marginLeft"
+        android:layout_marginTop="@dimen/sneaker_txtShoeName"
+        android:layout_toRightOf="@id/imgSneakerThumbnail"
+        android:textSize="@dimen/sneaker_shoeName"
+        android:textStyle="bold" />
+
+    <TextView
+        android:id="@+id/txtShoeDescription"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_alignLeft="@id/txtShoeName"
+        android:layout_alignTop="@id/txtShoeName"
+        android:layout_marginTop="@dimen/sneaker_marginTop"
+        android:text="@string/shoe_desc_text"
+        android:textSize="@dimen/sneaker_txtDesc_txtsz" />
```

Comment:
Put a layout_below attribute so that when the shoe name is too long it does not overlap with the description